control-plane/config/crd/bases/consul.hashicorp.com_meshes.yaml

```diff
@@ -62,10 +62,56 @@ spec:
+                              `false` instead of Consul's default value of `true`. This disables the normalization of request URL paths
+                              according to RFC 3986, as well as converting `\` to `/` and decoding non-reserved %-encoded characters.
+                              If using L7 intentions with path match rules, it is strongly recommended to enable path normalization in order to
+                              avoid match rule circumvention via non-normalized path values.
+                              The default value of this option is `false` (recommended).
+                            type: boolean
+                          mergeSlashes:
+                            description: |-
+                              MergeSlashes sets the value of the `merge_slashes` option in the Envoy listener's `HttpConnectionManager`.
+                              It controls the normalization of request URL paths by merging consecutive `/` characters (not part of RFC 3986).
+                              If using L7 intentions with path match rules, it is recommended that this setting be enabled to avoid match rule
+                              circumvention via non-normalized path values, unless legitimate service traffic depends on allowing for repeat
+                              `/` characters or upstream services are configured to differentiate between single and multiple slashes.
+                              The default value of this option is `false`.
+                            type: boolean
+                          pathWithEscapedSlashesAction:
+                            description: |-
+                              PathWithEscapedSlashesAction sets the value of the `path_with_escaped_slashes_action` option in the Envoy
+                              listener's `HttpConnectionManager`.
+                              It controls the action taken in response to request URL paths with escaped slashes in the path.
+                              If using L7 intentions with path match rules, it is recommended that this be configured to avoid match rule
+                              circumvention via non-normalized path values, unless legitimate service traffic depends on allowing for escaped
+                              `/` or `\` characters or upstream services are configured to differentiate between escaped and unescaped slashes.
+                              The default value of this option is empty, which is equivalent to `IMPLEMENTATION_SPECIFIC_DEFAULT`. See Envoy
+                              docs for more information on available options.
+                            type: string
+                        type: object
+                    type: object
                   sanitizeXForwardedClientCert:
                     type: boolean
-                required:
-                - sanitizeXForwardedClientCert
```

Comment:
Changed by adding `omitempty` to this field; now that we have multiple fields, requiring this one no longer makes sense. I don't know of any validation we require that a config entry has _some_ explicit fields (`service-defaults` is allowed to be empty), so I think just removing this is fine.